freqtrade/freqtradebot.py

```diff
@@ -756,23 +755,28 @@ def _check_and_execute_sell(self, trade: Trade, sell_rate: float,
             return True
         return False
 
+    def _check_timed_out(self, side: str, order: dict) -> bool:
+        """
+        Check if timeout is active, and if the order is still open and timed out
+        """
+        timeout = self.config.get('unfilledtimeout', {}).get(side)
```

Comment:
No need to use `.get()` here (see above, as in `_process()`, the check was removed)?

Btw, why are those `unfilledtimeout` settings marked as required? The default values that are now in the default config, can be efficiently applied to these settings.

Reply:
> No need to use .get() here (see above, as in _process(), the check was removed)?

you're right, I've overseen this check (so it was present twice, actually).

It's important to always call `check_handle_timedout` - since it checks 2 things - timeout - and for orders cancelled on exchange. Without this, we will not be able to detect this case.